hasura/migrations/default/1662391842396_interaction_events/down.sql

```diff
@@ -0,0 +1,47 @@
+
+alter table "public"."interaction_events" rename to "user_events";
```

Comment:
lets just clean this up to be one single migration rather than a create and a rename

Reply:
this is cleaned up